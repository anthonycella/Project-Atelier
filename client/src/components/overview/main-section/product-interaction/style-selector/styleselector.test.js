import renderer from 'react-test-renderer';
import StyleSelector from './styleselector.jsx';

it('Style Selector component renders to the page', () => {
  const component = renderer.create(
    <StyleSelector />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
