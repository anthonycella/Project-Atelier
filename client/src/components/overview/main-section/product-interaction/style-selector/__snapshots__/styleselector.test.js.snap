// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Style Selector component renders to the page 1`] = `
<div
  className="sc-iBkjds hLGONQ"
>
  <div
    className="sc-bczRLJ iNXLB"
  >
    <strong>
      Style 
      &gt;
       
    </strong>
    Selected Style
  </div>
  <div
    className="sc-gKXOVf dtTTxt"
  >
    <div
      className="sc-gsnTZi gFrKuk"
      onClick={[Function]}
    />
    <div
      className="sc-gsnTZi gFrKuk"
      onClick={[Function]}
    />
    <div
      className="sc-eCYdqJ fpYimD"
    >
      <div
        className="sc-hKMtZM sc-jSMfEi joHSMs dDeSWZ"
      />
      <div
        className="sc-dkzDqf dOpEzC"
      />
    </div>
    <div
      className="sc-gsnTZi gFrKuk"
      onClick={[Function]}
    />
    <div
      className="sc-gsnTZi gFrKuk"
      onClick={[Function]}
    />
    <div
      className="sc-gsnTZi gFrKuk"
      onClick={[Function]}
    />
    <div
      className="sc-gsnTZi gFrKuk"
      onClick={[Function]}
    />
    <div
      className="sc-gsnTZi gFrKuk"
      onClick={[Function]}
    />
  </div>
</div>
`;
